src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -1142,35 +1016,25 @@ void testNotErrorReportedDuringRegistrationWhenWhoIsRegisteringFieldIsSetToCorre
     void testNotErrorReportedDuringRegistrationForMissingWhoIsRegisteringFieldForFullValidationAndIsRedisRemovalEnabledFeatureFlagSetToTrueForPartialValidation() throws ServiceException
     {
         setIsRedisRemovalEnabledFeatureFlag(true);
-        buildPartialOverseasEntityUpdateSubmissionDto();
+        buildOverseasEntitySubmissionDto();
         overseasEntitySubmissionDto.setWhoIsRegistering(null);
 
         Errors errors = overseasEntitySubmissionDtoValidator.validatePartial(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, PASS_THROUGH_HEADER);
         assertFalse(errors.hasErrors());
     }
 
-    @Test
-    void testErrorReportedDuringRegistrationWhenWhoIsRegisteringFieldSetToWrongValueForFullValidationAndIsRedisRemovalEnabledFeatureFlagSetToTrueForPartialValidation() throws ServiceException
-    {
-        try {
-            setIsRedisRemovalEnabledFeatureFlag(true);
-            buildPartialOverseasEntityUpdateSubmissionDto();
-            overseasEntitySubmissionDto.setWhoIsRegistering("wrong_value");
-
-        } catch(Exception e) {
-            assertEquals("No enum constant uk.gov.companieshouse.overseasentitiesapi.model.WhoIsRegisteringType.WRONG_VALUE", e.getMessage());
-        }
-    }
 
     @Test
-    void testNotErrorReportedDuringRegistrationForMissingWhoIsRegisteringFieldForFullValidationAndIsRedisRemovalEnabledFeatureFlagSetToFalseForPartialValidation() throws ServiceException
+    void testErrorReportedDuringRegistrationWhenWhoIsRegisteringFieldSetToWrongValueForFullValidationAndIsRedisRemovalEnabledFeatureFlagSetToTrueForPartialValidation() throws ServiceException
```

Comment:
test name references FullValidation but I think it should be PartialValidation (test uses validatePartial)

Reply:
I will update the name